src/redux/slices/appPreferences.slice.ts

```diff
@@ -13,39 +15,63 @@ export const streamGranularityInSeconds = {
   hour: 3600,
   day: 86400,
   week: 86400 * 7,
-  month: 86400 * 30
-}
+  month: 86400 * 30,
+};
 
-type etherDecimalPlaces = 18 | 9 | 5; 
+type etherDecimalPlaces = 18 | 9 | 5;
 
 const initialState: IAppPreferences = {
-  themePreference: 'system',
-  streamGranularity: 'day',
-  etherDecimalPlaces: 18
+  themePreference: "system",
+  streamGranularity: "day",
+  etherDecimalPlaces: 18,
+  displayedTestNets: networksByTestAndName
```

Comment:
Because this is persisted, will it break with old state?

Reply:
as it's just an extension of the old state and not an alteration, I don't think

Reply:
It's `undefined` in the old state which could cause issues.

Try this:
1. Checkout to `master`
2. Delete the state (local storage & cookie)
3. Create new state based on `master`
4. Checkout to this branch
5. Check the developer console for errors when navigating around

Reply:
based on the flow you described, everything seems to be in order, no errors or visible problems.

Reply:
Gotcha